src/transform/src/predicate_pushdown.rs

```diff
@@ -281,6 +281,17 @@ impl PredicatePushdown {
                                     {
                                         push_down.push(localized);
                                         pushed = true;
+                                    } else if let Some(consequence) = input_mapper
+                                        .consequence_for_input(
+                                            predicate.clone(),
+                                            index,
+                                            &equivalences[..],
```

Comment:
Non need for `[..]`, it'll dereference to `&[Vec<_>]` without the range expression.

Reply:
Fixed